custom_components/adaptive_lighting/switch.py

```diff
@@ -1165,43 +1164,45 @@ async def _update_manual_control_and_maybe_adapt(
             context.id,
         )
 
-        if adapt_brightness is None:
-            adapt_brightness = self.adapt_brightness_switch.is_on
-        if adapt_color is None:
-            adapt_color = self.adapt_color_switch.is_on
+        adapt_brightness = adapt_brightness or self.adapt_brightness_switch.is_on
```

Comment:
Yeah I tried this change and ended up with errors in my test. This is your old code pulled from `_adapt_lights`